rails/es6.md

```diff
@@ -0,0 +1,105 @@
+## [Home](../README.md)
+
+## Summary
+
+Including ES6 in Rails applications can be very useful. Using JavaScript
+packages that are only available/maintained in npm version has traditionally
+been a problem with Rails application. This documents an approach chosen among a
+couple of different options.
+
+We settled upon sprockets-commoner over webpack and browserify because it
```

Comment:
I'm not opposed to that. I did include a References section at the bottom. If you think it should be included here in the Summary, I'm cool with that too.